src/internal/observable/connectable.ts

```diff
@@ -0,0 +1,47 @@
+/** @prettier */
+
+import { ObservableInput } from '../types';
+import { Subject } from '../Subject';
+import { Subscription } from '../Subscription';
+import { Observable } from '../Observable';
+import { defer } from './defer';
+
+export type ConnectableObservableLike<T> = Observable<T> & {
```

Comment:
nitpick: this could be an interface as per [TS team recommendation](https://github.com/microsoft/TypeScript/wiki/Performance#preferring-interfaces-over-intersections):

```ts
export interface ConnectableObservableLike<T> extends Observable<T> {
  connect(): Subscription;
}
```